### DISCUSSION TITLE
Compatibility with netbox 3.5.x ?

### DISCUSSION THREAD

**Discussion by greinick** [Onemind-Services-LLC/netbox-secrets#55] [Q&A] [answered]
Hi,  I'd like to know if you have any information about the compatibility with netbox 3.5.x ?

Thanks and regards.

**Comment by abhi1693** [comment]
There is a release planned for today or latest by tomorrow for adding support for v3.5.0.

**Comment by abhi1693** [comment, marked as answer]
:loudspeaker: NetBox Secrets v1.8.0 is now available! https://github.com/Onemind-Services-LLC/netbox-secrets/releases/tag/v1.8.0
